src/components/Network.tsx

```diff
@@ -0,0 +1,113 @@
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import { Box, Divider, IconButton } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import KeyboardReturnIcon from '@material-ui/icons/KeyboardReturn';
+import React from 'react';
+
+import { useSourceTarget } from '../contexts/SourceTargetContextProvider';
+import useDashboard from '../hooks/useDashboard';
+import { ChainDetails } from '../types/sourceTargetTypes';
+
+// As this is placed as a child in the Material UI Select component, for some reason style components classes are not working.
+// This way to inject the styles works.
+const useStyles = makeStyles((theme) => ({
+  main: {
+    position: 'relative',
+    ...theme.typography.h4,
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: theme.spacing(0.5),
+    '& .MuiIconButton-root': {
+      backgroundColor: theme.palette.background.paper,
+      position: 'absolute',
+      top: 0,
+      bottom: 0,
+      right: '-1rem',
+      width: '2rem',
+      height: '2rem',
+      margin: 'auto',
+      border: `1px solid ${theme.palette.divider}`,
+      transform: 'rotate(-90deg)',
+      transition: 'transform .15s',
+      '&:hover': {
+        transform: 'rotate(90deg)',
+        '& svg': {
+          transform: 'translateY(4px)',
+          transition: 'transform .1s',
+          '&.onlyHover': {
+            opacity: 1,
+            transform: 'scale(-1) translateY(4px)'
+          }
+        }
+      },
+      '& svg.onlyHover': {
+        opacity: 0,
+        position: 'absolute',
+        transform: 'scale(-1)'
+      }
+    }
+  },
+  statsEntry: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    fontSize: theme.typography.subtitle2.fontSize,
+    paddingLeft: 1,
+    paddingRight: 2,
+    '& span': {
+      ...theme.typography.subtitle2
+    }
+  }
+}));
+
+export const NetworkSides = () => {
+  const classes = useStyles();
+  const { sourceChainDetails, targetChainDetails } = useSourceTarget();
+  return (
+    <Box marginY={2} className={classes.main}>
+      <Box p>{sourceChainDetails.sourceChain}</Box>
+      <Divider />
+      <IconButton size="small">
+        <KeyboardReturnIcon className="onlyHover" fontSize="small" />
+        <KeyboardReturnIcon fontSize="small" />
+      </IconButton>
+      <Box p>{targetChainDetails.targetChain}</Box>
+    </Box>
+  );
+};
+
+export const NetworkStats = () => {
+  const classes = useStyles();
+  const dbSource = useDashboard(ChainDetails.SOURCE);
```

Comment:
may be i'm not following correctly the design ... but are we not going to show target chain details as well? In case we need to:  `NetworkStats` should receive `chain` as a prop and then call the component twice like:

```
< NetworkStats  chain={ChainDetails.SOURCE} />
< NetworkStats  chain={ChainDetails.TARGET} />
```

In case this is not what you have design , you can dismiss this comment

Reply:
@hbulgarini I do not see when the "target to source" stats might be useful, what actions this data might inform. So I would not show it. But I'm open to be proven wrong now or by user feedback/requests in the future :) However pls consider merging this first and create an issue to add the stats for the opposite direction from the state of UI.